modules/ROOT/pages/test-recorder.adoc

```diff
@@ -5,10 +5,41 @@ endif::[]
 
 The test recorder enables you to record a processing flow and then configure a unit test based on the captured event.
 
-The test recorder is an open beta feature that is compatible with Anypoint Studio 7.x.x and later and MUnit 2.2.1 and later.
+The test recorder captures real data as it goes through your application in Studio. By capturing the flow execution, MUnit can automatically generate an MUnit test so that you can configure the necessary mocks and assertions for your test. See xref:test-recorder-examples.adoc[Recording a Test from an Exchange Example] for a full working example.
+
+[NOTE]
+The test recorder is an open beta feature.
+
+== Compatibility
+
+The following table describes the required product versions to use the test recorder. These versions are for the beta feature and are likely to change for the GA (general availability) release.
+
+[%header%autowidth.spread,cols="a,a"]
+|===
+| Product | Version
+| Anypoint Studio
+| 7.4.0 and later.
+
+| MUnit
+| 2.2.4
```

Comment:
This is the runtime version of MUnit, but what you need to get is the `MUnit Studio Plugin` version 2.4.2, which will come bundled with the `MUnit Runtime` 2.2.4 and the `MUnit Test Recorder` 1.0.0.

Reply:
Then I'd suggest adding MUnit Anypoint Studio Plugin 2.4.2 to this list. (this is how it shows in the update site)